readme.md

```diff
@@ -70,6 +70,10 @@ Below you will find projects that have created/labelled various issues for Hackt
 
 - [first-timers-only](https://github.com/search?q=label%3Afirst-timers-only+state%3Aopen+type%3Aissue)
 - [for-new-contributors](https://github.com/search?q=label%3Afor-new-contributors+state%3Aopen+type%3Aissue)
+- [good-first-issue](https://github.com/search?q=label%3Agood-first-issue+state%3Aopen+type%3Aissue)
+- [hacktoberfest](https://github.com/search?q=label%3Ahacktoberfeststate%3Aopen+type%3Aissue)
```

Comment:
The link here is missing a `+`: `https://github.com/search?q=label%3Ahacktoberfest+state%3Aopen+type%3Aissue`

Reply:
Fixed. Sorry for the effort caused!